tests/queries/0_stateless/02373_progress_contain_result.sh

```diff
@@ -5,5 +5,5 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 . "$CURDIR"/../shell_config.sh
 
 echo 'SELECT 1 FROM numbers(100)' |
-  ${CLICKHOUSE_CURL_COMMAND} -v "${CLICKHOUSE_URL}&wait_end_of_query=1&send_progress_in_http_headers=0" --data-binary @- 2>&1 |
+  ${CLICKHOUSE_CURL_COMMAND} -vs "${CLICKHOUSE_URL}&wait_end_of_query=1&send_progress_in_http_headers=0" --data-binary @- 2>&1 |
```

Comment:
When you pass `-s` (--silent) option to curl, you should always also specify `-S` (not ignore errors).